tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/LiferayServerModuleLabelDecorator.java

```diff
@@ -98,4 +115,10 @@ private Image getBundleModuleImage()
 
         return image;
     }
+
+    private Image getErrorImage()
+    {
+        return ImageResource.getImage( ImageResource.IMG_PROJECT_MISSING );
```

Comment:
Error image should not be "IMG_PROJECT_MISSING" we should use the standard "ERROR" marker that you see in so many other places.  Also teh error marker should be lower left-hand corner instead of lower right hand corner.

![screen shot 2016-08-09 at 10 06 13 am](https://cloud.githubusercontent.com/assets/595221/17521515/466b1b06-5e19-11e6-8ff7-164c183da69a.png)

Also we should add text in the status of the module in the servers view, not just have the error marker only.  we need to give the user some indication as why the error marker is there.